charts/fint-adapter-sikri/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: fint-adapter-sikri
+description: A Helm chart for the sikri adapters
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+#appVersion: "3.9.0"
```

Comment:
For application using the FINT model, the app version should be the FINT version. You can find this in the `apiVersion` in the `gradle.properties` file in the project. E.g. https://github.com/FINTLabs/fint-sikri-arkiv-adapter/blob/f283d0cc8300d182f45a51c40023f58e2d5d9d94/gradle.properties#L2